contrib/review-trailers-tooling/README.md

```diff
@@ -0,0 +1,41 @@
+## What's this?
+
+A basic set of tools to ease adding review trailers, in order to acknowledge reviewers' work.
+
+## Which trailers are to be added?
+
+- `Helped-by: Full Name <email@address>`
+   Added by regular users to particular commits within a PR. Applies to the commit it appears on. Means "this commit was somehow improved based on suggestions by @reviewer".
+- `Reviewed-by: Full Name <email@address>`
+   Added by maintainer to PR merge commit. Applies to whole PR. Means "these changes were carefully reviewed by @reviewer an he has stated conformity with them."
```

Comment:
No, no, you're completely right. I'm talking about workflow here because what tools should do can depend on what the intended workflow is. But, when we reach an agreement on that, workflow description should be moved to wiki, as you mention.